pmm-app/tests/QAN/steps/qanActions.js

```diff
@@ -158,6 +157,27 @@ module.exports = {
     }
   },
 
+  applyFilterInSection(section, filter) {
```

Comment:
Because of duplicated values in different sections. For example, I want to use "postgres" filter to apply. It will show you 3 filters that contain the value. I wanted to prevent failings in the future if new filters would be added. :)